fix static build

Fix the following static build failure:

/home/buildroot/autobuild/instance-3/output-1/host/bin/sh4-buildroot-linux-musl-gcc -o fxload ezusb.o main.o
/home/buildroot/autobuild/instance-3/output-1/host/lib/gcc/sh4-buildroot-linux-musl/10.3.0/../../../../sh4-buildroot-linux-musl/bin/ld: /home/buildroot/autobuild/instance-3/output-1/host/lib/gcc/sh4-buildroot-linux-musl/10.3.0/libgcc.a(unwind-dw2.o): in function `size_of_encoded_value':
/home/buildroot/autobuild/instance-3/output-1/build/host-gcc-final-10.3.0/build/sh4-buildroot-linux-musl/libgcc/../../../libgcc/unwind-pe.h:89: undefined reference to `abort'

Fixes:
 - http://autobuild.buildroot.org/results/bca28d7a6d2b324fb61fe99b8af4b86caa2350ee

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

diff -Naur fxload-2008_10_13.orig/Makefile fxload-2008_10_13/Makefile
--- fxload-2008_10_13.orig/Makefile	2022-06-11 22:11:02.845916977 +0200
+++ fxload-2008_10_13/Makefile	2022-06-11 22:12:25.118006070 +0200
@@ -39,7 +39,7 @@
 
 # object files
 $(PROG): $(FILES_OBJ)
-	$(CC) -o $(PROG) $(FILES_OBJ)
+	$(CC) $(LDFLAGS) -o $(PROG) $(FILES_OBJ)
 
 %.o: %.c
 	$(CC) -c $(CFLAGS)  $< -o $@
